Tests/UnitTests/Scripts/tests.ts

```diff
@@ -617,12 +633,13 @@ describe("URLSearchParams", function () {
     const paramsSet = new URLSearchParams("");
 
     it("should throw exception when trying to set with less than 2 parameters", function () {
+        // @ts-expect-error
```

Comment:
Maybe add a comment on what the expected error is and why we need this?

Reply:
Fixed.